tests/unit/lms/services/canvas_api_test.py

```diff
@@ -251,9 +196,9 @@ def test_list_files(self, api_client, http_session):
         response = api_client.list_files("COURSE_ID")
 
         assert response == files
-        AnyRequest.assert_on_comparison = True
```

Comment:
This was some debugging stuff left in. This is still present in the new matcher, but currently private as `_assert_on_comparison`.